src/drawer.test.miscellaneous.ts

```diff
@@ -0,0 +1,274 @@
+import { html } from 'lit';
+import { expect, test } from './playwright/test.js';
+
+test('defines itself', { tag: '@miscellaneous' }, async ({ mount, page }) => {
+  await mount(
+    () => html`<glide-core-drawer label="Label">Content</glide-core-drawer>`,
+  );
+
+  const host = page.locator('glide-core-drawer');
+
+  await expect(host).toBeDefined('glide-core-drawer');
+});
+
+test(
+  'can be opened',
+  { tag: '@miscellaneous' },
+  async ({ browserName, mount, page, setProperty }) => {
+    test.skip(
+      browserName === 'webkit',
+      '"toggle" is dispatched inconsistently in CI. Or perhaps on Ubuntu. Try removing this when Webkit 26 is stable (TODO).',
+    );
+
+    await mount(
+      () => html`<glide-core-drawer label="Label">Content</glide-core-drawer>`,
+    );
+
+    const host = page.locator('glide-core-drawer');
+    const component = page.getByRole('complementary');
+
+    await expect(host).toDispatchEvents(
+      () => setProperty(host, 'open', true),
+      [
+        {
+          bubbles: true,
+          composed: true,
+          type: 'toggle',
+        },
+      ],
+    );
+
+    await expect(component).toBeVisible();
+  },
+);
+
+test(
+  'can be closed',
+  { tag: '@miscellaneous' },
+  async ({ browserName, mount, page, setProperty }) => {
+    test.skip(
+      browserName === 'webkit',
+      '"toggle" is dispatched inconsistently in CI. Or perhaps on Ubuntu. Try removing this when Webkit 26 is stable (TODO).',
+    );
+
+    await mount(
+      () =>
+        html`<glide-core-drawer label="Label" open>Content</glide-core-drawer>`,
+    );
+
+    const host = page.locator('glide-core-drawer');
+    const component = page.getByRole('complementary');
+
+    await expect(host).toDispatchEvents(
+      () => setProperty(host, 'open', false),
+      [
+        {
+          bubbles: true,
+          composed: true,
+          type: 'toggle',
+        },
+      ],
+    );
+
+    await expect(component).toBeHidden();
+  },
+);
+
+test(
+  'does not dispatch a "toggle" event when already open and opened programmatically',
+  { tag: '@miscellaneous' },
+  async ({ browserName, mount, page, setProperty }) => {
+    test.skip(
+      browserName === 'webkit',
+      '"toggle" is dispatched inconsistently in CI. Or perhaps on Ubuntu. Try removing this when Webkit 26 is stable (TODO).',
+    );
```

Comment:
Because we do `.not.toDispatchEvents(...)`, these actually do pass. But because we know the toggle event in the above cases **doesn't** fire, it's a bit worrisome to me as they could be a false positive. So I'd prefer skipping these for now and revisiting with the latest version of Webkit when it's ready.